cmd/livegrep-github-reindex/main.go

```diff
@@ -81,6 +83,17 @@ func main() {
 
 	gh := github.NewClient(h)
 
+	if *flagApiBaseUrl != "" {
+		if (*flagApiBaseUrl)[len(*flagApiBaseUrl)-1:] != "/" {
```

Comment:
[`strings.HasSuffix`](https://golang.org/pkg/strings/#HasSuffix) would make this significantly less line-noise-y.